apps/sscs/demo/base/kustomization.yaml

```diff
@@ -3,4 +3,5 @@ kind: Kustomization
 resources:
   - ../../base
 patchesStrategicMerge:
-  - ../../sscs-core-frontend/demo.yaml
\ No newline at end of file
+  - ../../sscs-core-frontend/demo.yaml
+  - ../../sscs-tribunals-frontend/demo.yaml
```

Comment:
this app appears to be not kustomized in demo so this isn't needed...

Reply:
Removed